src/cookbook/design/themes.md

```diff
@@ -89,44 +136,51 @@ Theme(
 );
 ```
 
-### Extending the parent theme
+### Extend the parent theme
 
-Rather than overriding everything, it often makes sense to extend the parent
-theme. You can handle this by using the [`copyWith()`][] method.
+Instead of overriding everything, consider extending the parent theme.
+To extend a theme, use the [`copyWith()`][] method.
 
 <?code-excerpt "lib/theme.dart (ThemeCopyWith)"?>
 ```dart
 Theme(
-  // Find and extend the parent theme using `copyWith`. See the next
-  // section for more info on `Theme.of`.
-  data: Theme.of(context).copyWith(splashColor: Colors.yellow),
+  // Find and extend the parent theme using `copyWith`.
+  // To learn more, check out the next section on `Theme.of`.
+  data: Theme.of(context).copyWith(
+      // TRY THIS: Change the seedColor to "Colors.red" or
+      //           "Colors.blue".
+      colorScheme: ColorScheme.fromSeed(seedColor: Colors.pink)),
   child: const FloatingActionButton(
     onPressed: null,
     child: Icon(Icons.add),
   ),
 );
 ```
 
-## Using a Theme
+## Apply a Theme
 
-Now that you've defined a theme, use it within the widgets' `build()`
-methods by using the `Theme.of(context)` method.
+To apply your new theme, add the `Theme.of(context)` method within the
+widgets' `build()` methods.
 
-The `Theme.of(context)` method looks up the widget tree and returns
+The `Theme.of(context)` method looks up the widget tree and retrieves
 the nearest `Theme` in the tree. If you have a standalone
-`Theme` defined above your widget, that's returned.
-If not, the app's theme is returned.
+`Theme` as defined in the previous example, that's applied.
+If not, Flutter applies the app's theme.
 
 In fact, the `FloatingActionButton` uses this technique to find the
 `accentColor`.
 
-<?code-excerpt "lib/main.dart (Container)" replace="/^child: //g"?>
+<?code-excerpt "lib/theme.dart (Container)" replace="/^child: //g"?>
 ```dart
 Container(
-  color: Theme.of(context).colorScheme.secondary,
+  padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 12),
+  color: theme.colorScheme.primary,
```

Comment:
Since you switched the source examples to have `final theme = Theme.of(context)` at the beginning of the `build` methods, these snippets no longer make sense. According to the previous text, they're meant to introduce the `Theme.of` method, but now that is no longer visible.

Either switch the source snippet back to `Theme.of` or include the `final theme = Theme.of(context)` at the beginning of the excerpt.

Reply:
Agreed. Fixing.